openjdk/build.gradle

```diff
@@ -268,6 +268,22 @@ def addNativeTest(nativeClassifier) {
             executable = javaExecutable
             testClassesDir = testSourceSet.output.classesDir
 
+            // TODO(nmittler): Is there a way to copy all properties of the test task?
```

Comment:
Hmm, possibly. Do all objects have a `properties` field (i.e. `test.testLogging.properties`)? I think I'll punt on this for now ... we can follow up later.